common/src/main/java/com/wynntils/utils/mc/ComponentUtils.java

```diff
@@ -40,22 +41,7 @@ public static CodedString getCoded(Component component) {
 
     // Text without formatting codes "Test text"
     public static String getUnformatted(Component component) {
-        return CodedString.fromComponentIgnoringComponentStylesAndJustUsingFormattingCodes(component)
-                .getUnformattedString();
-    }
-
-    public static CodedString getCoded(String jsonString) {
-        MutableComponent component = Component.Serializer.fromJson(jsonString);
-        if (component == null) return CodedString.EMPTY;
-
-        return getCoded(component);
-    }
-
-    public static String getUnformatted(String jsonString) {
-        MutableComponent component = Component.Serializer.fromJson(jsonString);
-        if (component == null) return null;
-
-        return getUnformatted(component);
+        return StyledText.fromComponent(component).getString(PartStyle.StyleType.NONE);
```

Comment:
I am not sure if we should do this. Maybe we should just leave this be until the whole method is removed